server/app/main.py

```diff
@@ -43,4 +55,6 @@ async def health_check():
 # Include the routers
 app.include_router(patient_router.app, prefix="/patients", tags=["patients"])
 app.include_router(doctor_router.app, prefix="/doctors", tags=["Doctor"])
-app.include_router(payment_router.app, prefix="/payments", tags=["Payments"])
\ No newline at end of file
+app.include_router(chat_router.router, prefix="/chat", tags=["chat"])
```

Comment:
Nit : tags=["Chat"]

Reply:
is there a reason to remove /payments router?

Reply:
payments is not removed, its still there

Reply:
> Nit : tags=["Chat"]

how does "chat" or "Chat" here make a difference?